multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/CreateServiceStep.java

```diff
@@ -95,20 +92,10 @@ private void processServiceCreationFailure(ProcessContext context, CloudServiceI
                              ExceptionMessageTailMapper.map(configuration, CloudComponents.SERVICE_BROKERS, service.getLabel()));
     }
 
-    private void updateServiceMetadata(CloudServiceInstanceExtended serviceToProcess, CloudControllerClient client) {
-        ImmutableCloudServiceInstance serviceWithMetadata = ImmutableCloudServiceInstance.copyOf(serviceToProcess);
-        CloudMetadata serviceMeta = client.getServiceInstance(serviceWithMetadata.getName())
-                                          .getMetadata();
-        serviceWithMetadata = serviceWithMetadata.withMetadata(serviceMeta);
-        client.updateServiceInstanceMetadata(serviceWithMetadata.getMetadata()
-                                                                .getGuid(),
-                                             serviceWithMetadata.getV3Metadata());
-    }
-
     @Override
     protected List<AsyncExecution> getAsyncStepExecutions(ProcessContext context) {
-        return Collections.singletonList(new PollServiceCreateOrUpdateOperationsExecution(getServiceOperationGetter(),
-                                                                                          getServiceProgressReporter()));
+        return Arrays.asList(new PollServiceCreateOrUpdateOperationsExecution(getServiceOperationGetter(), getServiceProgressReporter()),
```

Comment:
We should ensure that it will be last on the list of other executions somehow.
I dont have perfect solutions but there are two options:
- unit test
- short comment